aten/src/ATen/native/Blas.cpp

```diff
@@ -0,0 +1,63 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/native/Blas.h>
+#include <ATen/NamedTensorUtils.h>
+
+namespace at { namespace native {
+
+DEFINE_DISPATCH(addmv_stub);
+
+Tensor &addmv_out(Tensor& result, const Tensor &self_, const Tensor &mat, const Tensor &vec, Scalar beta, Scalar alpha) {
+  { // scope of NoNamesGuard
+
+  at::NoNamesGuard guard;
+  auto result_sizes = std::vector<int64_t> {mat.size(0)};
+  if (!result.defined()) {
```

Comment:
I think since result is passed by reference and we want the final result in the tensor passed by the user, the result tensor should not be undefined and so we can remove this check

Reply:
fixed